### DISCUSSION TITLE
Google Palm API

### DISCUSSION THREAD

**Discussion by xiao2c** [GaiZhenbiao/ChuanhuChatGPT#846] [Q&A] [answered]
config.json 中有google palm api key。但是哪里用到了palm api？

**Comment by Keldos-Li** [comment]
很抱歉！之前我们在测试分支上尝试引入PaLM，但是后面中止了，我后来拆分分支的时候漏掉了 d32517d42b562170a1ca2304e7751f872d973b3e 中对config中的修改，把这个配置项遗留了下来……目前，川虎Chat还没正式支持PaLM。

**Reply by Keldos-Li** [reply, marked as answer]
@xiao2c 现在已经支持palm，可以去试试~
